openedx_events/data.py

```diff
@@ -28,16 +39,26 @@ class EventsMetadata:
         minorversion (int): version of the event type.
         source (str): logical source of an event.
         sourcehost (str): physical source of the event.
-        time (datetime): timestamp when the event was sent.
-        sourcelib (str): Open edX Events library version.
+        time (datetime): (optional) timestamp when the event was sent with
+            UTC timezone. Defaults to current time in UTC. See OEP-41 for
+            details.
+        sourcelib (tuple of ints): Open edX Events library version. A tuple was
+            selected so that version comparisons don't have to worry about
+            lexical ordering of strings (e.g. '0.9.0' vs. '0.10.0').
```

Comment:
@rgraber: This is in its own commit, but sorry about adding it to this PR. The original type was wrong, and I wasn't sure why we chose tuples. The comment comes from https://github.com/openedx/openedx-events/pull/7#discussion_r667294019.